### ISSUE TITLE
Sammy's "Zcash-Proxy" for BTCGPU fails to connect to bgoldd - Sammy fix your software

### ISSUE THREAD

**Issue by bilbobits** [BTCGPU/BTCGPU#138]
https://github.com/sammy007/zcash-proxy

equihash proxy for btg-bitcoin-gold aka fen-coin  ( fen means shit in mandarin chinese, pronounced 'fun', like its fun to scam white idiots )

***
CLI, note everything is correct

./zcash-proxy.linux config.json2017/11/05 08:04:14 Version: devfee-408cc6c
2017/11/05 08:04:14 Loading config: /home/Mining/btcgpu/zcash-proxy/config.json
2017/11/05 08:04:14 Upstream: btgproxy => http://127.0.0.1:9332
2017/11/05 08:04:14 Default upstream: btgproxy => http://127.0.0.1:9332
2017/11/05 08:04:14 Error while refreshing info on btgproxy: 401 Unauthorized
2017/11/05 08:04:15 Unable to validate coinbase address: 401 Unauthorized

***
debug diag from bgoldd ( btcgpu daemon ), note that daemon is saying incorrect credentials sent,
 
2017-11-05 01:04:14 Received a POST request for / from 127.0.0.1:33990
2017-11-05 01:04:14 ThreadRPCServer incorrect password attempt from 127.0.0.1:33

***

Sammy U got this bitch dialed in for dev-fee 2.0, U ain't updated it since Jan 2017, it would probably take you 5 minutes to support the fact that BTCGPU doesn't have 'getblocksubsidy', and you could fix this verification problem.

I see you a lot here bitching to dev's about them not 'fixing' their software, now the ball is in your court, and of course you will get paid, you will get 2% of nada :)

***

The handshaking here is that most likely zcash-proxy isn't getting a response for 'getblocksubsidy', thus the entire proxy fails, bgoldd daemon doesn't complete the hand-shaking and calls it off as a loging verification failure. Sammy can easily just add glue that say's that when 'getblocksubsidy' fails cuz the CLI doesn't support it, then just faux an ALL-ZERO data structure and be happy, as of now btcgpu (fen-coin) doesn't support subsidy, some cold day in hell maybe the developers will add 'getblocksubsidy' to btc-gpu and return an all-zero block, until then sammy must step up and patch the dike.

**Comment by bilbobits** [comment]
If this binary were open-source, then this problem could/would have been fixed weeks ago, but its not open source, which is why it doesn't work.

Lastly the dev's since day one have said "BTCGPU doesn't support solo mining", to this we can all say Fuck the Dev's and what they want, I personally find all the mining pools including "pool.gold" ( owned by these devs )  to be SCAMS, and nicehash being the worst of all, its like all the scammers in china have discovered 'mining pools'

**Comment by sammy007** [comment]
> If this binary were open-source, then this problem could/would have been fixed weeks ago, but its not open source, which is why it doesn't work.

Sorry I am not here to make you rich, I am here to make rich myself. Zcash-proxy is for zcash and of course I already got mod for BTCGPU. This is total offtopic in this issue tracker.

It's a total bullshit about "does not support solo mining". I already modified my software for BTCGPU mining. Solo mining is pool mining if you are running pool for yourself.

@h4x3rotab pls close it.
